requirements.txt

```diff
@@ -1,7 +1,6 @@
 speech2text
 text2speech>=0.1.9
 psutil
-jarbas_hive_mind>=0.10.3
```

Comment:
jarbas_utils needs to be migrated to ovos_utils in hivemind-core and this version bumped